content/blog/2023/03/03/2023-03-09-jira-donation.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Jira for the Jenkins project"
+tags:
+- issues
+- jira
+authors:
+- markewaite
+opengraph:
+image: /images/post-images/2023/03/09/2023-03-09-jira-for-jenkins/jira-for-jenkins.png
+discourse: true
+---
+
+= Jira continues for Jenkins development
+
+The Jenkins project has used link:https://www.atlassian.com/software/jira[Jira Software] for issue tracking since 2005.
+Jira has been our issue and enhancement tracking system for almost as long as Jenkins core has existed.
+Jenkins core and most Jenkins plugins track issues and enhancements through a link:https://issues.jenkins.io[Jira] instance hosted for the Jenkins project by the Linux Foundation.
+
+image::/images/post-images/2023/03/09/2023-03-09-jira-for-jenkins/jira-for-jenkins.png[Atlassian sponsors Jenkins]
+
+We're pleased to announce that Atlassian has agreed to continue sponsoring the Jenkins project with a license to use Jira.
+We're deeply grateful to Atlassian for their support of open source software and especially for their support of the Jenkins project.
+We extend our thanks to the Linux Foundation as well for their continued hosting of Jira for the Jenkins project.
+
+== Upgrade schedule
+
+As part of the sponsorship, the Linux Foundation will need to update Jira and restart it.
+The upgrade will happen beginning at midnight, Saturday March 11, 2023 UTC (4:00 PM Seattle time on Friday March 10, 2023).
+The system may be down for as much as 30 minutes.
+
+== Jenkins and Jira by the numbers
+
+The Jenkins project Jira installation includes over 70,000 issues collected over the course of 18 years.
+Some examples of the breadth and depth in the issue tracker include:
+
+* 350+ epics, including 5 phases of link:https://issues.jenkins.io/browse/JENKINS-52012[Java 11 transition], 3 phases of link:https://issues.jenkins.io/browse/JENKINS-67908[Java 17 transition], and various improvement projects like link:https://issues.jenkins.io/browse/JENKINS-68446[JAXB], link:https://issues.jenkins.io/browse/JENKINS-69083[Jakarta Mail], link:https://issues.jenkins.io/browse/JENKINS-69988[HttpClient], and  link:https://issues.jenkins.io/browse/JENKINS-70301[WMI Windows agents]
+* Over 2000 components
```

Comment:
Didn't you mention that number as well in the meeting?

* Over 20000 users

Reply:
> Didn't you mention that number as well in the meeting?
> 
> * Over 20000 users

Just 20k? There are 137746 accounts registered on Jira :P

Reply:
I could insert "Over 130000 registered users" as an additional data point.

Reply:
Added in 503ba0bc4061d98fc61ede995d2e8297a9a909fe